docs/sidebar.nightly.template.yml

```diff
@@ -0,0 +1,63 @@
+index: index.md
```

Comment:
This file is exactly old `sidebar.yml` with reference sections removed (+ additional comment)